src/azure-cli/azure/cli/command_modules/backup/tests/latest/test_backup_commands.py

```diff
@@ -1317,6 +1317,7 @@ def test_backup_identity(self, resource_group, vault_name):
     @VaultPreparer(parameter_name='vault1')
     @VaultPreparer(parameter_name='vault2')
     @KeyVaultPreparer(additional_params='--enable-rbac-authorization false')
+    @record_only()
```

Comment:
May I ask why do you need to mark this test as `@record_only()` ?

Reply:
In order to be run the test in recorded/live mode, we have to comment/uncomment some lines that come up later on. Since manual intervention has to be made to run the test in live mode, I've added `@record_only()`

Reply:
My question is why do you want this test skip the live mode testing?

Reply:
I do run the live mode testing whenever an API version upgrade happens. If it seems unnecessary though I can remove the `@record_only`, should I do that?

Reply:
After adding the `@record_only` flag, executing live mode will skip this test

Reply:
Ah I see what you meant. I'll revert this bit of code.